kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -49,6 +49,7 @@
 import org.kie.server.api.model.ServiceResponse;
 import org.kie.server.api.model.ServiceResponsesList;
 import org.kie.server.api.model.instance.SolverInstance;
+import org.optaplanner.persistence.xstream.api.score.AbstractScoreXStreamConverter;
 import org.optaplanner.persistence.xstream.api.score.buildin.bendable.BendableScoreXStreamConverter;
```

Comment:
yes, they are - but we do "optimize imports" just before releasing CR1 normally (not all module owners currently do this, but I think we should).
Unlike other style changes, I tend to completely ignore unused imports as they are autofixed later anyway, as I think they're a waste of time to fix manually.

Just before CR1, I also tend to check findbugs, intellij inspect, etc - always worth it.